plugins/peermanager/main.go

```diff
@@ -0,0 +1,108 @@
+package peermanager
+
+import (
+	"fmt"
+
+	"github.com/Shopify/sarama"
+	"github.com/ethereum/go-ethereum/log"
+	"github.com/ethereum/go-ethereum/node"
+	"github.com/openrelayxyz/xplugeth"
+	"github.com/openrelayxyz/xplugeth/hooks/initialize"
+	"github.com/openrelayxyz/xplugeth/types"
+)
+
+type peerManagerConfig struct {
+	BrokerURL string `yaml:"peerManager.broker.url"`
```

Comment:
Let's make the yaml `broker.url`. Since plugeth2 will have its own config files for each plugin, we don't need to namespace it here too.